api/dbadapters/bigquery.ts

```diff
@@ -223,43 +226,41 @@ export class BigQueryDbAdapter implements IDbAdapter {
         tableMetadataMap.set(action.target, await this.getMetadata(action.target));
       })
     );
-    const updateQueries = actions.map(action => {
-      const definitionHash = hashTableDefinition(action);
-      const dependencies = action.dependencyTargets;
-      const metadata =
-        action.type === "operation"
-          ? { lastModifiedTime: `${new Date().valueOf()}` }
-          : tableMetadataMap.get(action.target);
-      const persistTable = dataform.PersistedTableMetadata.create({
-        target: action.target,
-        lastUpdatedMillis: Long.fromString(metadata.lastModifiedTime),
-        definitionHash,
-        dependencies
-      });
+    const queries = actions
```

Comment:
IMO all of the code that produces `queries` belongs inside `run_cache.ts`

Reply:
This code will be different for different warehouse. I have moved the selectQuery part to run_cache